File: examples/spork/working/both_modified_change_parameter_varargs/Expected.java
class Cls {
    public static void add(int... b) {
        int sum = 0;
        for (int val : b) {
            sum += val;
        }
<<<<<<< LEFT
        return a + sum;
||||||| BASE
        return a + b;
=======
        return sum;
>>>>>>> RIGHT
    }
}

File: examples/cpp/working/unable_to_detect_disjoint_conditions/Expected.c
#include<stdio.h>

int main(int argc, char** argv) {
    if (argc < 2) {
        printf("too few arguments\n");
        exit(1);
    }
<<<<<<< LEFT
    if (argc == 3) {
        printf("hello\n");
    }
||||||| BASE
=======
    if (argc == 4) {
        printf("world\n");
    }
>>>>>>> RIGHT
}
